### PR TITLE
BETA: Register kkll.is-a.dev

### PR DESCRIPTION
**Pull request by K-K-L-L**
Added `kkll.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/kkll.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "K-K-L-L",
+        "email": "kkll345643@protonmail.com"
+    },
+    "record": {
+        "CNAME": "is-a.dev"
```
Comment (CuteDog5695) [inline domains/kkll.json:7]: please put a valid record, this wont work

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/kkll.json (hunk 1 above)
Comment (K-K-L-L) [reply domains/kkll.json:7]: mb i forgot is-a.dev hosting shut down recently

Comment (CuteDog5695) [reply domains/kkll.json:7]: but even if is-a.dev hosting was still available it would still be wrong lmfao

Comment (K-K-L-L) [reply domains/kkll.json:7]: lol

**Pull request closed without merging**